types/knockout/index.d.ts

```diff
@@ -4,23 +4,22 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 
-interface KnockoutSubscribableFunctions<T> {
-    [key: string]: KnockoutBindingHandler | undefined;
-
-	notifySubscribers(valueToWrite?: T, event?: string): void;
+interface KnockoutExtensionFunctions {
+    [key: string]: any;
 }
 
-interface KnockoutComputedFunctions<T> {
-    [key: string]: KnockoutBindingHandler | undefined;
+interface KnockoutSubscribableFunctions<T> extends KnockoutExtensionFunctions {
+    notifySubscribers(valueToWrite?: T, event?: string): void;
 }
 
-interface KnockoutObservableFunctions<T> {
-    [key: string]: KnockoutBindingHandler | undefined;
+interface KnockoutComputedFunctions<T> extends KnockoutExtensionFunctions {
```

Comment:
I'm wondering whether interface `KnockoutExtensionFunctions` Is needed at all. I have only included it because it is in keeping with the original code.